lifx-core/src/lib.rs

```diff
@@ -1164,6 +1164,22 @@ pub enum Message {
     /// Message type 507
     GetMultiZoneEffect,
 
+    /// Message type 508
+    SetMultiZoneEffect {
+        /// The unique value identifying this effect
+        instance_id: u32,
+        typ: MultiZoneEffectType,
+        reserved: u16,
+        /// The time it takes for one cycle of the effect in milliseconds
+        speed: u32,
+        /// The amount of time left in the current effect in nanoseconds
+        duration: u64,
+        reserved7: u32,
+        reserved8: u32,
+        /// The parameters that was used in the request.
+        parameters: [u8; 32],
```

Comment:
The [lifx docs](https://lan.developer.lifx.com/docs/changing-a-device#setmultizoneeffect---packet-508) say this:

> This field is **8** `4` byte fields which change meaning based on the effect that is running.

I'm wondering if it makes sense to write this field as a `[u32; 8]` ?

Reply:
Yeah, that would make more sense  Ithink.

Reply:
Would you want parameters in StateMultiZoneEffect updated to match?